abdm_integrator/hip/models.py

```diff
@@ -11,3 +16,43 @@ class ConsentArtefact(models.Model):
 
     class Meta:
         app_label = 'abdm_hip'
+
+
+class LinkRequestDetails(models.Model):
+    link_reference = models.UUIDField(default=uuid.uuid4, unique=True)
+    patient_reference = models.CharField(max_length=255)
+    patient_display = models.TextField()
+    hip_id = models.CharField(max_length=255)
+    status = models.CharField(choices=LinkRequestStatus.CHOICES, default=LinkRequestStatus.PENDING,
+                              max_length=40)
+    initiated_by = models.CharField(choices=LinkRequestInitiator.CHOICES, max_length=40)
+    error = models.JSONField(null=True)
+    date_created = models.DateTimeField(auto_now_add=True)
+    last_modified = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        app_label = 'abdm_hip'
+        indexes = [
+            models.Index(fields=['patient_reference', 'hip_id'])
+        ]
+
+
+class LinkCareContext(models.Model):
+    reference = models.CharField(max_length=255)
+    display = models.TextField()
+    health_info_types = models.JSONField(default=list)
+    additional_info = models.JSONField(null=True)
+    link_request_details = models.ForeignKey(LinkRequestDetails, on_delete=models.PROTECT,
+                                             related_name='care_contexts')
```

Comment:
I couldn't locate where you are actually making use of this related_name.

Reply:
Added it for in case of future need and avoids migrations in case we need to add. Also useful for debug.